src/components/SettingsBlock/SettingsBlock.test.js

```diff
@@ -0,0 +1,101 @@
+import SettingsBlock from '.';
+import {
+  fireEvent,
+  render,
+  screen,
+} from '../../utils/tests/renderWithRouter';
+
+const onClick = jest.fn();
+
+const rendered = (
+  isActive = false,
+  isBoolean = false,
+  icon = 'user',
+  children = 'Settings'
+) =>
+  render(
+    <SettingsBlock
+      onClick={onClick}
+      isActive={isActive}
+      isBoolean={isBoolean}
+      icon={icon}
+    >
+      {children}
+    </SettingsBlock>
+  );
+describe('SettingsBlock Component test suits', () => {
+  it('Should render correctly with its default values', () => {
+    rendered();
+    expect(screen.getByText('Settings')).toBeInTheDocument();
+    expect(screen.getByText('Settings').parentElement).toHaveClass(
+      'group w-full flex items-center justify-between gap-[25px] p-3 bg-white hover:bg-primary-200 cursor-pointer text-gray-500 text-xl font-semibold rounded-lg'
```

Comment:
Place this string inside of a const

Reply:
Ok I'll in this moment!!! Thanks for recomendation!